src/style.css

```
@@ -1 +1,59 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  padding: 10px 50px;
```

```suggestion
  padding: 0 50px;
```
Сверху не нужны паддинги, задавай нужную высоту внутренним элементам

---

done